src/components/Toolbar/Index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+    StyleSheet,
+    ToolbarAndroid,
+    ToolbarAndroidAction
+} from 'react-native';
+import * as React from 'react';
+import icons, {
+    images
+} from '../../assets/index';
+
+/*Only one button is permitted on the left
+it must either be a menu button which has a 'burger' icon
+or a back button which has an arrow icon*/
```

Comment:
I think it is good practice to have comments that are descriptive enough to explain what it is for, but doesn't have to go into specifics about the possible options (especially because there could be scenarios in the feature where the comment may no longer apply)

Reply:
I have removed the description of the icon

Reply:
I would recommend using a more formal documentation method (such as jsdoc)